pkgs/top-level/perl-packages.nix

```
@@ -2487,6 +2500,21 @@ let
     };
   };
 
+  ChordPro = buildPerlPackage {
+    pname = "App-Music-ChordPro";
+    version = "0.974";
+    src = fetchurl {
+      url = mirror://cpan/authors/id/J/JV/JV/App-Music-ChordPro-0.974.tar.gz;
+      sha256 = "7fae50535dcb7f68171a8c1f8ad49b13563f406fb1d44eba78b349e7ef83a8af";
+    };
+    propagatedBuildInputs = [ AppPackager FileLoadLines FontTTF IOString ImageInfo PDFAPI2 StringInterpolateNamed ];
```

```suggestion
    buildInputs = [ PodParser ];
    propagatedBuildInputs = [ AppPackager FileLoadLines IOString ImageInfo PDFAPI2 StringInterpolateNamed TextLayout Wx ];
```
`PodParser` is needed for compatibility with `perl-5.32.0`
`Wx` can be added for `bin/wxchordpro` support
`FontTTF` is already included by `PDFAPI2`
`TextLayout` is also needed (by 0.977), here is the derivation:

```nix
  TextLayout = buildPerlPackage {
    pname = "Text-Layout";
    version = "0.019";
    src = fetchurl {
      url = "mirror://cpan/authors/id/J/JV/JV/Text-Layout-0.019.tar.gz";
      sha256 = "a043f2a89e113b29c523a9efa71fa8398ed75edd482193901b38d08dd4a4108e";
    };
    buildInputs = [ PDFAPI2 ];
    meta = {
      description = "Pango style markup formatting";
      license = with stdenv.lib.licenses; [ artistic1 gpl1Plus ];
    };
  };
```